examples/make-deploy/README.md

```diff
@@ -59,7 +59,11 @@ make letsencrypt-notebook
 
 The first command creates a Docker volume named after the notebook container with a `-secrets` suffix. It then runs the `letsencrypt` client with a slew of options (one of which has you automatically agreeing to the Let's Encrypt Terms of Service, see the Makefile). The second command mounts the secrets volume and configures Jupyter to use the full-chain certificate and private key.
 
-Be aware: Let's Encrypt has a pretty [low rate limit per domain](https://community.letsencrypt.org/t/public-beta-rate-limits/4772/3) at the moment. Don't exhaust your requests playing around!
+Be aware: Let's Encrypt has a pretty [low rate limit per domain](https://community.letsencrypt.org/t/public-beta-rate-limits/4772/3) at the moment.  To help avoid exhausting your limit, `make letsencrypt` defaults to hitting the Let's Encrypt staging servers.  To hit their production servers, set the `CERT_SERVER` environment variable to a blank string.
+
+```
+make letsencrypt FQDN=host.mydomain.com EMAIL=myemail@somewhere.com CERT_SERVER=''
```

Comment:
If --staging is the default, then the default command just above needs to include the CERT_SERVER=''.

IMHO, the default should be production, not staging, since the main use case here will be "I just want to run it and get a cert" not "I'd like to develop these scripts and don't want to run out of requests."